docs/open-source-project/release-notes/tizen-5-5-m1.md

```diff
@@ -0,0 +1,631 @@
+  - MediaPlayer
+    - SetPlayPosition() block issue with MediaStreamSource has been fixed.
+- Multimedia Daemon
+  - Murphy
+    - Fix the launching failure issue with PIDFile.
+  - Mm-resource-manager
+    - Fix the issue of GDBus method related with usb detachable.
+  - Muse-server
+    - Fix the busy loop issue caused for high CPU usage.
+
+
+## Network and Connectivity
+
+### New and changed features
+
+- User awareness framework
+  - A framework for user detection based on multiple sensors has been added.
+- Bluetooth A2DP multipoint connection support
+  - A2DP sink target can support up-to 2 different source devices.
+- Wi-Fi band steering
+  - Band steering mechanism between 2.4GHz and 5GHz of the same AP has been added.
+- API changes
+  - Intelligent network monitoring
+    - API to get network interface link information has been added.
+    - API to test network connection status (availability of gateway and DNS server) has been added.
+    - API to test reachability of HTTP server has been added.
+  - Wi-Fi manager
+    - API to get whether 5GHz band is supported has been added.
+    - API set to get EAP anonymous identity has been added.
+  - Bluetooth
+    - Privilege level of OOB data getter API has been changed.
+  - Smart traffic control
+    - API to clone and destroy statistics info handle has been added.
+    - Legacy API to get statistics info asynchronously has been deprecated.
+    - API to get all statistics info has been added.
+    - Legacy API to get total statistics info has been deprecated.
+  - curl version up (7.62.0)
+  - C# API
+    - BluetoothLeDevice API set getting the remote device information has been added.
+    - Legacy C# Gatt API has been deprecated.
+    - New BluetoothGattClient C# API has been added.
+    - BluetoothLeDevice API set getting the remote device information has been deprecated.
+    - Bluetooth - Wrong AcceptStateChangedEventArgs class property and DestroyServerSocket API has been deprecated.
+    - Bluetooth - New SocketConnection class property has been added.
+    - Bluetooth - API to destroy Gatt server has been added.
+    - STC C# API has been added.
+    - API to check Wi-Fi scan state has been added.
+    - API to check cellular roaming state has been added.
+    - Wi-Fi - Exceptions to provide accurate error information has been added.
+
+
+## Security
+
+### New and changed features
+
+- Security-manager
+  - Smack rules will be loaded directly from security-manager DB; all rule files have been removed.
+- Privilege
+  - The following privileges have been added.
+    - autofillmanager, internal/buxton/systemsetting, filesystem.read, filesystem.write, windowsystem.admin, d2d.datasharing, d2d.remotelaunch
+- Privilege-info
+  - New API has been added.
+    - Getting several privilege information as list.
+
+
+## Service framework
+
+### New and changed features
+
+- Account -Manager
+  - Added feature to query account information even after account deletion.
+
+### Fixes
```

Comment:
From line 448 to 468, please change it to the "has been" format.

Reply:
will accept